buildscripts/cstor-csi-driver/Dockerfile

```diff
@@ -1,25 +1,39 @@
+# Copyright 2020 The OpenEBS Authors. All rights reserved.
 #
-# This Dockerfile builds a recent volume-mgmt using the latest binary from
-# volume-mgmt  releases.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
 #
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
 
 FROM ubuntu:16.04
 RUN apt-get update; exit 0
 RUN apt-get -y install rsyslog xfsprogs
 #RUN apt-get clean && rm -rf /var/lib/apt/lists/*
 
+ARG ARCH
```

Comment:
Are we using this argument?

Reply:
not at the moment. We need to use this in the name label, when we refactor the builds again to build for different architectures.